cmd/snap/cmd_connect.go

```diff
@@ -27,8 +27,8 @@ import (
 
 type cmdConnect struct {
 	Positionals struct {
-		Offer SnapAndName `required:"yes"`
-		Use   SnapAndName
+		Use   SnapAndName `required:"yes"`
```

Comment:
I was thinking about `PlugRef` and `SlotRef` even if `PlugRef` and refer to just a snap name and both can refer to the core snap implicitly. Perhaps `PlugSpec` or something like this?